services/user/Workshop/query/src/lib.rs

```diff
@@ -0,0 +1,37 @@
+#[psibase::service]
+#[allow(non_snake_case)]
+mod service {
+    use workshop;
+    use async_graphql::*;
+    use psibase::*;
+
+    pub struct Query;
+
+    #[Object]
+    impl Query {
+        /// Get the app metadata for a specific app ID
+        async fn app_metadata(
+            &self,
+            account_id: AccountNumber,
+        ) -> Option<workshop::service::AppMetadata> {
+            workshop::service::AppMetadataTable::new()
+                .get_index_pk()
+                .get(&account_id)
+        }
+
+        async fn event(
```

Comment:
You probably saw & copied this `event` query function somewhere else, but it's not very useful. I think we can probably delete this (and eventually remove it from everywhere else, too).